structcol/refractive_index.py

```diff
@@ -158,9 +159,202 @@ def n_silica_colloidal(w):
 def n_water(w):
     return 1.33
 
-def n_cargille(i,series,wavelength):
-    # where i is the cardinal number of the liquid (starting with 0) and w is the wavelength, in nm (converted to um)
-    return 1.45
+def n_cargille(i,series,w):
```

Comment:
Probably we should leave it the way it is for now.  We don't need to force it into the current data structure if it doesn't fit, but we should add the tests.